vignettes/articles/enteric_fever_example.Rmd

```diff
@@ -103,21 +106,12 @@ Column Name | Description
 ```{r data, message = FALSE}
 
 #Import cross-sectional data from OSF and rename required variables
-xs_data <- 
-  "https://osf.io/download//n6cp3/" %>% 
-  load_pop_data() 
+xs_data <- "https://osf.io/download//n6cp3/" %>%
+  load_pop_data()
```

Comment:
let's replace `load_pop_data()` with  `readr::read_rds() %>% as_pop_data()` which is a more typical workflow (see #148)